### PR TITLE
Adding CAP blog post

### PR DESCRIPTION
**Pull request by clare-stanton**
Blog post about CAP's data embargo being lifted, and the site transitions

### CODE REVIEW

File: app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: Transitions for the Caselaw Access Project
+guest-author: The Library Innovation Lab Team
+excerpt_separator: <!--more-->
+---
+The team at the Library Innovation Lab is excited to announce the original limitations on the data available for the [Caselaw Access Project](http://case.law) expired this month, and that data can now be fully released without restriction on access or use.
```
Comment (bensteinberg) [inline app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md]:
```suggestion
The Library Innovation Lab is excited to announce that the original limitations on the data available for the [Caselaw Access Project](http://case.law) expired this month, and that data can now be fully released without restriction on access or use.
```

File: app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+As part of our original collaboration agreement with Ravel Law, Inc. (now part of Lexis-Nexis) for the Caselaw Access Project there had been access limitations on the full text and bulk data available, which have now expired. Over the next few months, we will be partnering with other organizations in the open legal data space like the [Free Law Project](http://free.law) to shepherd this data into its next phase.
```
Comment (bensteinberg) [inline app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md]:
```suggestion
As part of our original agreement with Ravel Law, Inc. (now part of Lexis-Nexis), there had been access limitations on the full text and bulk data available, which have now expired. Over the next few months, we will be partnering with other organizations in the open legal data space like the [Free Law Project](http://free.law) to shepherd this data into its next phase. 
```

File: app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+## History of the Caselaw Access Project 
+
+In 2018, the Library Innovation Lab [launched case.law](https://www.nytimes.com/2015/10/29/us/harvard-law-library-sacrifices-a-trove-for-the-sake-of-a-free-database.html) to host and distribute data created by the Caselaw Access Project. Its release was the culmination of several years of work at the Harvard Law Library to digitize a corpus of 6 million cases representing almost all precedential law in the United States. The cases were digitized from Harvard’s own collection of hardbound court reporters from across the nation, an archive which predates the founding of the United States. The digitization process involved removing the bindings of each reporter, scanning 40 million pages, and using OCR technology to convert the PDF images into human and machine readable text. You can see parts of that process in this [video we released about the project](https://vimeo.com/922493882?share=copy).
```
Comment (bensteinberg) [inline app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md]:
```suggestion
In 2018, the Library Innovation Lab [launched case.law](https://www.nytimes.com/2015/10/29/us/harvard-law-library-sacrifices-a-trove-for-the-sake-of-a-free-database.html) to host and distribute data created by the Caselaw Access Project. Its release was the culmination of several years of work at the Harvard Law Library to digitize a corpus of 6 million cases representing almost all precedential law in the United States. The cases were digitized from Harvard’s own collection of hardbound court reporters from across the nation, an archive which predates the founding of the United States. The digitization process involved removing the binding of each volume, scanning 40 million pages, and using OCR technology to convert the PDF images into human and machine-readable text. You can see parts of that process in this [video we released about the project](https://vimeo.com/922493882?share=copy). 
```

**Review by bensteinberg — COMMENTED** [review]
Some copyediting suggestions

File: app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md (hunk 2 above)
Comment (jcushman) [inline app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md]:
```suggestion
As part of our original collaboration agreement with Ravel Law, Inc. (now part of LexisNexis) for the Caselaw Access Project there had been access limitations on the full text and bulk data available, which have now expired. Over the next few months, we will be partnering with other organizations in the open legal data space like the [Free Law Project](http://free.law) to shepherd this data into its next phase. The Free Law Project already includes all CAP cases, as well as cases scraped from court websites, in its [CourtListener](https://www.courtlistener.com/) search engine.
```

File: app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+---
+title: Transitions for the Caselaw Access Project
+guest-author: The Library Innovation Lab Team
+excerpt_separator: <!--more-->
+---
+The team at the Library Innovation Lab is excited to announce the original limitations on the data available for the [Caselaw Access Project](http://case.law) expired this month, and that data can now be fully released without restriction on access or use.
+
+As part of our original collaboration agreement with Ravel Law, Inc. (now part of Lexis-Nexis) for the Caselaw Access Project there had been access limitations on the full text and bulk data available, which have now expired. Over the next few months, we will be partnering with other organizations in the open legal data space like the [Free Law Project](http://free.law) to shepherd this data into its next phase. 
+
+As the case.law website will no longer need to serve as the primary access point for the data, we will be **winding down services that our team has provided since the initial release of CAP in 2018, including the API. **
```
Comment (jcushman) [inline app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md]:
```suggestion
We will continue hosting the CAP data in bulk for researchers, and as individual readable cases, at case.law. However, **we will be winding down services that can be better provided elsewhere, such as the search function and API.**
```

File: app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+<!--more-->
+
+We have transitioned the original CAP website into an archival state. The previous version of the site, along with a selection of its tools, will still be available at [old.case.law](http://old.case.law) until September of this year.
```
Comment (jcushman) [inline app/_posts/2024-03-26-transitions-for-the-caselaw-access-project.md]:
```suggestion
The previous version of the site will still be available at [old.case.law](http://old.case.law) until September of this year. If there are features of the previous site that are not well covered by the current site or by CourtListener, we welcome feedback to info@case.law.
```

**Review by jcushman — APPROVED** [review]

**Pull request merged**